src/plugins/unified_search/public/autocomplete/providers/value_suggestion_provider.test.ts

```diff
@@ -95,7 +100,7 @@ describe('FieldSuggestions', () => {
 
     it('should otherwise request suggestions', async () => {
       const [field] = stubFields.filter(
-        ({ type, aggregatable }) => type === 'string' && aggregatable
+        ({ type, aggregatable }) => (type === 'string' || type === 'ip') && aggregatable
```

Comment:
Would it be helpful to also call `await getValueSuggestions({...})` 2 times then? And check that `http.fetch` was called twice.

Reply:
Addressed in 0e2c81f1fa04f99702fc5bfd927ab913d2b78009.